mesop/BUILD

```diff
@@ -15,7 +15,10 @@ py_library(
     srcs = [
         "__init__.py",
     ],
-    data = glob([".env"]),
+    data = glob([
+        ".env",
+        "static/**/*",
```

Comment:
Not sure if this is the best way to add static folder support for bazel dev, especially in the downstream scenario.

Reply:
Yeah, this one is slightly annoying because we have to manually set the folder to mesop/static. I tried to move the static folder down to the root level, but I think I didn't set up the BUILD file right so that it was included in the runfiles.